sw/host/hsmtool/src/commands/aes/import.rs

```diff
@@ -54,14 +54,17 @@ impl Dispatch for Import {
         let secret = Secret::Aes;
         let key = helper::read_file(&self.filename)?;
         if self.unwrap.is_some() {
+            if self.unwrap_mechanism.is_none() {
+                return Err(anyhow!("Missing flag: unwrap-mechanism"));
+            }
```

Comment:
I prefer to use `ok_or` or `ok_or_else` for this kind of situation.  Instead of the extra `if/return` construct, you can transform the `Option` into `Result` and short-circuit out with the `?` operator:

```
            let _object = secret.unwrap_key(
                session,
                result.id.clone(),
                result.label.clone(),
                key,
                self.template.clone(),
                self.unwrap.as_deref(),
                self.unwrap_mechanism.as_ref().ok_or(anyhow!("Missing flag ..."))?,
            )?;
```

Reply:
Fixed.